### DISCUSSION TITLE
No chat access in the multiplayer room

### DISCUSSION THREAD

**Discussion by tiseppw** [ppy/osu#16098] [Q&A] [answered]
**Describe the bug:** If you create / join a multiplayer game and then close the chat in the room via the chat list (`F8`), it will be impossible to communicate with other players.

**Version:** osu!lazer 2021.1211.0.0

https://user-images.githubusercontent.com/33760265/146154801-d7612241-a0ff-40b5-8cd1-22107695925d.mp4

**Comment by frenzibyte** [comment, marked as answer]
Escalated to #16099 and resolved in #16109.
